analytics/core.go

```diff
@@ -47,17 +49,21 @@ type AmpObject struct {
 	Origin               string
 	StartTime            time.Time
 	HookExecutionOutcome []hookexecution.StageOutcome
+	SeatNonBid           []openrtb_ext.SeatNonBid    `json:"-"`
+	RequestWrapper       *openrtb_ext.RequestWrapper `json:"-"`
 }
 
 // Loggable object of a transaction at /openrtb2/video endpoint
 type VideoObject struct {
-	Status        int
-	Errors        []error
-	Request       *openrtb2.BidRequest
-	Response      *openrtb2.BidResponse
-	VideoRequest  *openrtb_ext.BidRequestVideo
-	VideoResponse *openrtb_ext.BidResponseVideo
-	StartTime     time.Time
+	Status         int
+	Errors         []error
+	Request        *openrtb2.BidRequest
+	Response       *openrtb2.BidResponse
+	VideoRequest   *openrtb_ext.BidRequestVideo
+	VideoResponse  *openrtb_ext.BidResponseVideo
+	StartTime      time.Time
+	SeatNonBid     []openrtb_ext.SeatNonBid    `json:"-"`
+	RequestWrapper *openrtb_ext.RequestWrapper `json:"-"`
```

Comment:
What is the purpose of these json paths tied to these variables

Reply:
I am unable to follow the question but, 

1. These fields are ignored from marshalling / un-marshalling, with the thought that `PubStack` module is marshalling entire AuctionObject. So, avoiding marshalling of these two fields because - 
      a. It many increase the network packet size, as SeatNonBid will capture all non-positive bids, whereas `RequestWrapper` is just a helper for keeping the unmarshalled version of extensions

Reply:
What I'm wondering is why do we need ``json:"-"`` to be next to these fields if they are going to be ignored anyway? Can't the json part of it be deleted, and instead these lines just look like:

```
SeatNonBid     []openrtb_ext.SeatNonBid
RequestWrapper *openrtb_ext.RequestWrapper
```

Reply:
@AlexBVolcy : These fields will not be ignored by `PubStack` module, if `json:"-"` is not mentioned. 

> I am unable to follow the question but,
> 
> 1. These fields are ignored from marshalling / un-marshalling, with the thought that `PubStack` module is marshalling entire AuctionObject. So, avoiding marshalling of these two fields because -
>    a. It many increase the network packet size, as SeatNonBid will capture all non-positive bids, whereas `RequestWrapper` is just a helper for keeping the unmarshalled version of extensions